### ISSUE TITLE
Support for pure Java or Kotlin library modules

### ISSUE THREAD

**Issue by johnjohndoe** [NeoTech-Software/Android-Root-Coverage-Plugin#75]
I am trying to understand whether pure Java library modules and pure Kotlin library modules are supported.
In the [README](https://github.com/NeoTech-Software/Android-Root-Coverage-Plugin/blob/d4a5cb47583bec6f5a1d36ce43200452fb09e80a/README.md#2-how-to-use) I found this statement:

> Only Android modules (`com.android.application` or `com.android.library`) are supported, this plugin will not execute tests and generate coverage reports for non-android modules. 

In issue #41 @Rolf-Smit stated:

> This library does come with an integration test that contains a pure Java module, ...

On [Reddit](https://www.reddit.com/r/androiddev/comments/o37lh9/comment/h3f3ctr/?utm_source=share&utm_medium=web2x&context=3) @Rolf-Smit stated:

> That is correct, currently it will not create coverage tasks for pure Java/Kotlin modules.

What is the state of support for non-Android modules? If unsupported, are there plans to add support?

**Comment by Rolf-Smit** [comment]
Non-Android modules are as of now not supported. The fact that the integration test has a non-Android module is purely to make sure it doesn't break the plugin in anyway.

Plans to add support? Yes I would love to add support for pure Java and pure Kotlin modules, however I don't have time to work on this anytime soon.

**Comment by johnjohndoe** [comment]
I totally respect your time constraints. Just to leave a thought behind: I tried using [arturdm/jacoco-android-gradle-plugin](https://github.com/arturdm/jacoco-android-gradle-plugin) to configure a project. For the Java library modules I simply added:

``` groovy
apply plugin: "jacoco"

jacocoTestReport {
    dependsOn test
}
```

This added the desired `jacocoTestReport` task to the `verification` group. Not sure if it is that easy.
